src/components/DocumentList.tsx

```
@@ -13,104 +13,194 @@ import {
-import { CheckIcon, CloseIcon, InfoOutlineIcon } from "@chakra-ui/icons";
+import { CheckIcon, CloseIcon, InfoOutlineIcon, ViewIcon } from "@chakra-ui/icons";
 import Table from "./common/table/Table";
+import { decodeBase64ToJson, isBase64 } from "../services/helperService"
+import { omit } from "lodash";
 
-// Document interface
 export interface Document {
   id: number;
   type: string;
   title: string;
   content: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)